### ISSUE TITLE
Building with Cloudflare plugin is broken

### ISSUE THREAD

**Issue by riker09** [abiosoft/caddy-docker#195]
Or the documentation is outdated. When I do this:

```
docker build -t caddy --build-arg plugins=cors,realip,expires,cache,cloudflare github.com/abiosoft/caddy-docker.git
```

I get this error:

```
...
go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: finding cloud.google.com/go v0.34.0
go: error loading module requirements
error at 'building caddy'
The command '/bin/sh -c VERSION=${version} PLUGINS=${plugins} ENABLE_TELEMETRY=${enable_telemetry} /bin/sh /usr/bin/builder.sh' returned a non-zero code: 1
```

Building *without* the Cloudflare plugin works. 🤔

**Comment by abiosoft** [comment]
This has been fixed now. `docker rmi abiosoft/caddy:builder` and try again.

**Comment by riker09** [comment]
The funny thing about this was that I have a build pipeline established and it was failing for several weeks now. It was only for an uncritical system within our deployment, so I didn't gave it much attention. But when I did I tried building locally and got the same error. I tried pulling the image, of course, but although I could see a newer version in Docker Hub I wasn't able to pull it. `docker pull` said that I already had the latest version.

This was the reason why I opened this ticket and this is also the reason why I'm commenting on this closed issue. For the record. 🙂
